Cargo.toml

```diff
@@ -31,3 +32,4 @@ members = [
 # Using a version of rust-multihash that compiles on emscripten
 # TODO: remove once merged upstream
 "multihash" = { git = "https://github.com/tomaka/rust-multihash", branch = "emscripten-hack" }
+"tokio-timer" = { git = "https://github.com/tomaka/tokio-timer", branch = "emscripten-test" }
```

Comment:
This contains https://github.com/tokio-rs/tokio-timer/pull/41 plus a small hack.